selenoid/docker.go

```diff
@@ -686,6 +686,7 @@ func (c *DockerConfigurator) PrintArgs() error {
 const (
 	videoDirName = "video"
 	logsDirName  = "logs"
+	networkName  = "selenoid"
```

Comment:
```suggestion
	networkName  = "aerokube"
```
?

Reply:
I would leave Selenoid just because it would be better to separate Selenoid and other future tools.